apps/fhir/bluebutton/utils.py

```diff
@@ -204,6 +220,9 @@ def request_call(request, call_url, crosswalk=None, timeout=None, get_parameters
         cert = ()
 
     header_info = generate_info_headers(request)
+
```

Comment:
Is this the only place where we're making requests to the backend that require the `keep-alive` header?

Reply:
There is a call for the patient resource in medicare_cb.views but that doesn't have the problems that the EOB has.
The request_call and request_get_with_params are the only 2 calls to the backend.